src/Microsoft.Framework.PackageManager/Program.cs

```diff
@@ -68,8 +68,9 @@ public int Main(string[] args)
 
                 c.OnExecute(async () =>
                 {
-                    var command = new RestoreCommand(_environment);
+                    var command = new RestoreCommand();
                     command.Reports = CreateReports(optionVerbose.HasValue(), feedOptions.Quiet);
```

Comment:
We'll remove it in later refactoring. `Reports` is still used by some utils inside `RestoreCommand`